distribution/springboot_setup/README.md

```diff
@@ -132,13 +132,27 @@ camel.output.endpoint.file.location=/tmp/openmrs-dbsync/store
 
 Note: To disable this feature, just delete the route, or rename it to **directory-sync-route.xml.disable**
 
-### 5. Rebuild the project
+### 5. Rebuilding reciever's openmrs databse search index
+To configure rebuiding of the receiver's database search index when entities such as patient are synced, set the following properties appropriately.
```

Comment:
End the sentence with a `":"` not a `"."`, also typo on `"rebuiding"`. Anyway paste this:
```
To configure rebuilding the receiver's database search index when entities such as patient are synced, set the following properties appropriately:
```